modules/rgbd/include/opencv2/rgbd.hpp

```diff
@@ -379,6 +379,26 @@ namespace rgbd
     {
     }
 
+    /** Constructor
+     * @param block_size The size of the blocks to look at for a stable MSE
+     * @param min_size The minimum size of a cluster to be considered a plane
+     * @param threshold The maximum distance of a point from a plane to belong to it (in meters)
+     * @param sensor_error_a_ coefficient of the sensor error with respect to the. 0 by default, 0.0075 for a Kinect
+     * @param sensor_error_b_ coefficient of the sensor error with respect to the. 0 by default
+     * @param sensor_error_c_ coefficient of the sensor error with respect to the. 0 by default
+     * @param method The method to use to compute the planes, by default RGBD_PLANE_METHOD_DEFAULT
+     */
+    RgbdPlane(int block_size, int min_size, double threshold, double sensor_error_a_ = 0,
+              double sensor_error_b_ = 0, double sensor_error_c_ = 0, int method =
+              RgbdPlane::RGBD_PLANE_METHOD_DEFAULT);
```

Comment:
> int method = RgbdPlane::RGBD_PLANE_METHOD_DEFAULT

Please move to the beginning of parameters list (methods define meaning of its parameters)